nipype/interfaces/mrtrix3/tracking.py

```diff
@@ -262,7 +262,8 @@ class Tractography(MRTrix3Base):
     >>> tk.inputs.roi_mask = 'mask.nii.gz'
     >>> tk.inputs.seed_sphere = (80, 100, 70, 10)
     >>> tk.cmdline                               # doctest: +ELLIPSIS
-    'tckgen -algorithm iFOD2 -mask mask.nii.gz -seed_sphere \
+    'tckgen -algorithm iFOD2 -samples 4 -output_seeds out_seeds.nii.gz \
+-mask mask.nii.gz -seed_sphere \
```

Comment:
Just a note you can drop the backslashes from this output string.

Reply:
it doesn't work without